dynamiq/nodes/agents/react.py

```diff
@@ -924,50 +923,75 @@ def _run_agent(
 
         for loop_num in range(1, self.max_loops + 1):
             try:
-                llm_result = self._run_llm(
-                    messages=self._prompt.messages,
-                    tools=self._tools,
-                    response_format=self._response_format,
-                    config=config,
-                    **kwargs,
-                )
+                # Create a streaming callback
+                streaming_callback = None
+                original_streaming_enabled = self.llm.streaming.enabled
+
+                if self.streaming.enabled:
+                    streaming_callback = ReActAgentStreamingParserCallback(
```

Comment:
As far as I understand, we are always using this callback underhood and shouldn't specify in workflow run config?